components/Loading.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2021 The Rode Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import styles from "styles/modules/Loading.module.scss";
+import { useTheme } from "providers/theme";
+
+const Loading = () => {
```

Comment:
Could this component take a prop (maybe named `loading`/`isLoading`), and then return null if that's false? That would reduce checks like this: `loading && <Loading />`